server/src/main/java/org/apache/druid/server/coordinator/rules/Rule.java

```diff
@@ -51,6 +51,14 @@
 
   boolean appliesTo(Interval interval, DateTime referenceTimestamp);
 
+  /**
+   * Returns whether this Rules should be matched and considered in loadstatus API.
+   * In general, Rules that load segment onto any Druid node should return true.
+   * Any Rule that returns true for this method should add a compute logic (as if case condition) for the particular
+   * Rule class in {@link DruidCoordinator#computeUnderReplicationCountsPerDataSourcePerTierForSegments}
+   */
+  boolean matchLoadStatusCount();
```

Comment:
It seems leaky to tie this call to an API method, rather, what is being captured here is whether or not a `Rule` will load data instead of dropping data, because not all rules that load data are of type `LoadRule`. Should this just be `isLoadRule()` or something to that effect?

Reply:
I don't like calling it `isLoadRule` because there is a class/interface `LoadRule` and not all things that return true for this method are of the class/subclass of that class/interface `LoadRule`. But I agree let's not tie it to API. How about calling it `isRuleLoadSegments` or `canLoadSegments` or `isLoadSegments`.
Although, I still want to leave the comment `Any Rule that returns true for this method should add a compute logic (as if case condition) for the particular Rule class in {@link DruidCoordinator#computeUnderReplicationCountsPerDataSourcePerTierForSegments}` there. Since that will help people find the appropriate place to change/add when creating new Rule. What do you think?

Reply:
I think `canLoadSegments` works, also `doesLoadSegments` or `willLoadSegments`.

Reply:
Done